nginx.montagu.conf

```diff
@@ -70,16 +70,22 @@ server {
     location /contribution/ {
         proxy_pass http://contrib/;
     }
+
+    set $orderly_web http://orderly_web_web:8888;
     location /reports/ {
-        proxy_pass http://orderly_web_web:8888/;
-        proxy_redirect default;
+        resolver 127.0.0.11 valid=30s;
+        rewrite ^/reports/(.*) /$1 break;
+        proxy_pass $orderly_web;
+        proxy_redirect http://orderly_web_web:8888/ /reports/;
 
         location "~/reports/(?<name>[^/]+)/(?<version>\d{8}-\d{6}-[0-9a-f]{8})" {
             return 301 /reports/report/$name/$version;
         }
     }
     location /pull-request/ {
-        proxy_pass http://support.montagu.dide.ic.ac.uk:4567/pull-request/;
+        resolver 127.0.0.11 valid=30s;
```

Comment:
using the same technique here because the webook might not be up when UAT/science restart after a reboot